e2e/testcases/apiservice_test.go

```diff
@@ -35,14 +35,24 @@ const (
 func TestCreateAPIServiceAndEndpointInTheSameCommit(t *testing.T) {
 	nt := nomostest.New(t, ntopts.Unstructured, ntopts.RequireGKE(t))
 	t.Cleanup(func() {
+		nt.MustKubectl("delete", "-f", "../testdata/apiservice/rbac.yaml", "--ignore-not-found")
+		nt.MustKubectl("delete", "-f", "../testdata/apiservice/namespace.yaml", "--ignore-not-found")
 		if t.Failed() {
 			nt.PodLogs(adapterNamespace, adapterName, "pod-custom-metrics-stackdriver-adapter", true)
 		}
 	})
 	nt.T.Log("Creating commit with APIService and Deployment")
+	if nt.MultiRepo {
+		nt.RootRepos[configsync.RootSyncName].Copy("../testdata/apiservice/rbac.yaml", "acme/cluster/rbac.yaml")
+	} else {
+		// In mono repo mode the necessary ClusterRole and ClusterRolebinding
+		// in rbac.yaml is blocked from being removed from git repo by git-importer in
+		// sharedTestEnv, so manually deploying them ahead.
+		nt.MustKubectl("apply", "-f", "../testdata/apiservice/namespace.yaml")
+		nt.MustKubectl("apply", "-f", "../testdata/apiservice/rbac.yaml")
+	}
 	nt.RootRepos[configsync.RootSyncName].Copy("../testdata/apiservice/namespace.yaml", "acme/namespaces/custom-metrics/namespace.yaml")
```

Comment:
Does the committing of this namespace also need to be moved up to the `if` statement on line 46? It looks like this would get applied twice in monorepo mode (manually applied and committed to git repo)